### ISSUE TITLE
ENH: Fetching action for full Diamond database

### ISSUE THREAD

**Issue by Sann5** [bokulich-lab/q2-moshpit#97]
### Is your feature request related to a problem?
No. It would be nice to have an action that downloads the Diamond database for downstream Eggnog analyses. 

### Describe the solution you'd like
This could be done by simply calling the script and piping it with the appropriate yes/no answers to indicate which database should be downloaded. For example:

```bash
printf "n\nn\ny" | download_eggnog_data.py -s --daata_dir . 
```

### Additional considerations
- Care should be taken that the output artifact can be accessed in the downstream actions.
- Care should be taken an appropriate Semantic Types is used for the generated database artifact. See this issue bokulich-lab/q2-types-genomics#64

**Comment by misialq** [comment]
Hey @Sann5, thanks for the thorough investigation and description! Here are some thoughts:
- your second snippet shows fetching only one db (Bacteria) but it should be possible to specify multipleDBs and fetch them into the same artifact
- I don't think it's such a big deal that you would need to re-download; I think instead we could have another action which just allows using a custom FASTA file to build custom DBs but that's something we can explore after
- as an alternative to your option B, I think we could rather have an action to _merge_ existing DBs into one (by copying all the dmnd files into a single artifact of the same type

**Comment by misialq** [comment]
Blocked by https://github.com/bokulich-lab/q2-types-genomics/issues/64

**Comment by Sann5** [comment]
**Issue updated.** 
- Previously the issue described the need to create an action that was able to both fetch the entire Diamond BD and build costume Diamond DB for selected groups of organisms. 
- Now this issue is only concerned with the action to allow the fetching of the entire Diamond BD. 
- A separate issue will be created for the build-diamond-DB action.
